api/submission/routes.go

```diff
@@ -7,14 +7,15 @@ import (
 )
 
 var routes []*core.Route = []*core.Route{
-    core.NewAPIRoute(core.NewEndpoint(`submission/history`), HandleHistory),
-    core.NewAPIRoute(core.NewEndpoint(`submission/peek`), HandlePeek),
     core.NewAPIRoute(core.NewEndpoint(`submission/fetch/attempts`), HandleFetchAttempts),
+    core.NewAPIRoute(core.NewEndpoint(`submission/fetch/course-report`), HandleFetchCourseReport),
```

Comment:
We don't want to the first one, because we may add other things in the future that may be considered "course reports".
For example, we can have a report on a courses resource utilization (disk, mem, cpu, etc).
What we have here is more of a "course submissions report".
We won't rename it internally for now (we'll wait until we have other actual reports),
but we want to be more careful with externally facing things, since we can't change those without a breaking change.

Reply:
Should I relocate the files that I am working on to report/ or should I just change the API endpoint in routs?

Reply:
Yes, you will need to move the files.